qhub/template/{{ cookiecutter.repo_directory }}/infrastructure/gcp.tf

```diff
@@ -33,6 +33,18 @@ module "kubernetes" {
       instance_type = "{{ nodegroup_config.instance }}"
       min_size      = {{ nodegroup_config.min_nodes }}
       max_size      = {{ nodegroup_config.max_nodes }}
+    {% if cookiecutter.gpu_enabled is defined %}
+    {% if ((cookiecutter.gpu_enabled) * (nodegroup == "user")) == 1 %}
```

Comment:
Why is this defined this way? We want the user to be able to create arbitrary node groups with gpus in them. Not necessarily just "user". 

```
{% if cookiecutter.gpu_enabled is defined %}
{% if ((cookiecutter.gpu_enabled) * (nodegroup == "user")) == 1 %}
```

Can just be `{% if nodegroup_config.guest_accelerators is defined %}` and remove `gpu_enabled = True`

Reply:
yeah. You are right

Reply:
Motivation here is the user may possibly want a node group with small gpus and another with large expensive ones that require a larger cpu/ram.